tests/doc_comment.rs

```diff
@@ -0,0 +1,16 @@
+extern crate snafu;
+
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+enum Error {
+    /// No user available
```

Comment:
How does this look for a pathological docstring, such as

```rust
/// No user available. This is unlikely to occur.
///
/// Call me Ishmael, and lo the slithy toves did lorem ipsum…
```

Should we detect when two consecutive newlines have been provided and only take values before that?

Reply:
Yeah sounds like a good idea. I implemented this.